components/browser/engine-gecko-beta/README.md

```diff
@@ -12,6 +14,46 @@ Use gradle to download the library from JCenter:
 implementation "org.mozilla.components:engine-gecko-beta:{latest-version}
 ```
 
+### Initializing
+
+It is recommended t create only one `GeckoEngine` instance per app. To create a `GeckoEngine` a `GeckoRuntime` and optionally `DefaultSettings` are needed.
```

Comment:
Small improvement, the **t** missed the **o**
It is recommended **t** should be t**o**

Reply:
Thanks!